app/institutions/dashboard/-components/institutional-users-list/template.hbs

```diff
@@ -0,0 +1,68 @@
+<PowerSelect
+        @options={{this.departments}}
+        @searchEnabled={{false}}
+        @selected={{this.department}}
+        @triggerClass={{local-class 'select'}}
+        @onchange={{action 'onSelectChange'}} as |department|
+>
+    {{department}}
+</PowerSelect>
+
+<ContentPlaceholders as |placeholder|>
+    <PaginatedList::HasMany
+            local-class='table'
+            @modelTaskInstance={{this.model.taskInstance}}
+            @relationshipName='institutionalUsers'
+            @query={{this.query}} as |list|
+    >
+        <list.header local-class='header'>
+            {{#let (component 'sort-button'
+                class=(local-class 'sort-button')
+                sortAction=(action this.sortInstitutionalUsers)
+                sort=this.sort
+            ) as |SortButton|}}
+                <div data-test-header-name>
+                    <strong>{{t 'institutions.dashboard.users_list.name'}}</strong>
+                    <SortButton @sortBy='userFullName' />
+                </div>
+                <div data-test-header-department>
+                    <strong>{{t 'institutions.dashboard.users_list.department'}}</strong>
+                    <SortButton @sortBy='department' />
+                </div>
+                <div data-test-header-public-projects>
+                    <img src='/assets/images/institutions/open-lock.svg'
```

Comment:
These images are pretty generic. I think we should put them in `/assets/global`.